### PR TITLE
Ragini/internal public page improvements/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss [hunk 1]
```diff
@@ -71,8 +71,22 @@
...
     }
   }
+
+  @media screen and (max-width: 1280px) {
+    .main-container {
+      .form {
+        flex-direction: column;
+        gap: 20px;
+
+        div {
```
Comment (Saurabh186) [inline src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss:81]: ??

File: src/appPages/publicPage/publicPage.scss [hunk 2]
```diff
@@ -26,6 +26,9 @@
                         min-width: unset !important;
                         width: 100px;
                         padding: 6px !important;
+                        border:  2px solid #ced4da;
+                        background-color: white;
```
Comment (Saurabh186) [inline src/appPages/publicPage/publicPage.scss:30]: usse varibles

File: src/components/collectionVersions/collectionParentPages.jsx [hunk 3]
```diff
@@ -266,14 +266,14 @@ const CollectionParentPages = (props) => {
                   </div>
                   <div className='dropdown-menu dropdown-menu-right'>
                     <div className='dropdown-item d-flex align-items-center' onClick={() => openEditPageForm(pageId)}>
-                      <FiEdit2 color='gray' /> Rename
+                      <FiEdit2 size={16}  color='#383636'/> Rename
                     </div>
                     {isOrgDocType() && <div className='dropdown-item d-flex align-items-center' onClick={() => setShowVersionForm(true)}>
-                      <SlSettings color='gray' /> Manage Version
+                      <SlSettings size={16}  color='#383636'/> Manage Version
```
Comment (Saurabh186) [inline src/components/collectionVersions/collectionParentPages.jsx:272]: variables use nahi kar sakte kya??
by default size 16 rahta hai remove krke check kr lena ek bar

File: src/components/collectionVersions/collectionParentPages.jsx [hunk 4]
```diff
@@ -266,14 +266,14 @@ const CollectionParentPages = (props) => {
...
                     </div>}
                     <div className='dropdown-item d-flex align-items-center text-danger delete-parent-btn'
                       onClick={() => openDeletePageModal(pageId)}>
-                      <RiDeleteBin6Line size={15} /> Delete
+                      <RiDeleteBin6Line size={16} /> Delete
```
Comment (Saurabh186) [inline src/components/collectionVersions/collectionParentPages.jsx:276]: same

File: src/components/collections/collections.jsx [hunk 5]
```diff
@@ -214,11 +214,11 @@ const Collections = (props) => {
                       {!collections[collectionId]?.importedFromMarketPlace && (
                         <>
                           <div className='dropdown-item d-flex align-items-center' onClick={() => openEditCollectionForm(collectionId)}>
-                            <FiEdit2 color='gray' /> Rename
+                            <FiEdit2 color='#383636' size={16} /> Rename
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:217]: same

File: src/components/collections/collections.jsx [hunk 6]
```diff
@@ -214,11 +214,11 @@ const Collections = (props) => {
...
                           </div>
                           {collections[collectionId].isPublic && (
                             <div className='dropdown-item d-flex align-items-center' onClick={() => handleGoToDocs(collections[collectionId])}>
-                              <IoDocumentTextOutline color='gray' /> Go to API Documentation
+                              <IoDocumentTextOutline size={16} color='#383636' /> Go to API Documentation
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:221]: same

File: src/components/collections/collections.jsx [hunk 7]
```diff
@@ -227,20 +227,20 @@ const Collections = (props) => {
                               TagManagerModalOpen(collectionId)
                             }}
                           >
-                            <IoPricetagOutline color='gray' /> Add Google Tag Manager
+                            <IoPricetagOutline size={16} color='#383636'  /> Add Google Tag Manager
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:230]: ....

File: src/components/collections/collections.jsx [hunk 8]
```diff
@@ -227,20 +227,20 @@ const Collections = (props) => {
...
                           </div>
-                          {!getCurrentOrg()?.is_readable &&<div className='dropdown-item' onClick={() => handleOrgModalOpen(collections[collectionId])}>
-                            <RiShareForward2Line size={16} color='grey' /> Move
+                          {!getCurrentOrg()?.is_readable && <div className='dropdown-item' onClick={() => handleOrgModalOpen(collections[collectionId])}>
+                            <RiShareForward2Line size={16} color='#383636' /> Move
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:233]: ...

File: src/components/collections/collections.jsx [hunk 9]
```diff
@@ -227,20 +227,20 @@ const Collections = (props) => {
...
                           </div>}
                           <div className='dropdown-item d-flex' onClick={() => openRedirectionsPage(collections[collectionId])}>
-                            <TbDirections size={16} color='grey' /> Redirections
+                            <TbDirections size={16} color='#383636' /> Redirections
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:236]: ....

File: src/components/collections/collections.jsx [hunk 10]
```diff
@@ -227,20 +227,20 @@ const Collections = (props) => {
...
                           </div>
                           {isOrgDocType() && <div className='dropdown-item' onClick={() => handleApiAutomation(collectionId)}>
-                            <TbSettingsAutomation size={16} color='grey' />
+                            <TbSettingsAutomation size={16} color='#383636' />
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:239]: ....

File: src/components/collections/collections.jsx [hunk 11]
```diff
@@ -227,20 +227,20 @@ const Collections = (props) => {
...
                             API Automation
                           </div>}
                           {isOrgDocType() && <div className='dropdown-item d-flex align-items-center h-auto'>
-                            <BiExport className='mb-1' size={18} color='grey' />
+                            <BiExport className='mb-1' size={16} color='#383636' />
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:243]: .....

File: src/components/collections/collections.jsx [hunk 12]
```diff
@@ -251,7 +251,7 @@ const Collections = (props) => {
                             className='text-danger dropdown-item delete-button-sb d-flex align-items-center delete-collection-btn'
                             onClick={() => openDeleteCollectionModal(collectionId)}
                           >
-                            <RiDeleteBin6Line size={12} /> Delete
+                            <RiDeleteBin6Line size={16} /> Delete
```
Comment (Saurabh186) [inline src/components/collections/collections.jsx:254]: ....

File: src/components/endpoints/displayEndpoint.jsx [hunk 13]
```diff
@@ -3222,7 +3222,7 @@ class DisplayEndpoint extends Component {
         style={this.state.themes.backgroundStyle}
       >
         <div className={this.isNotDashboardOrDocView() ? 'mainContentWrapper dashboardPage' : 'mainContentWrapper d-flex'}>
-          <div className={`innerContainer w-100 response-bottom ${!isDashboardRoute(this.props) && 'd-flex justify-content-around'}`}>
+          <div className={`innerContainer response-bottom w-100 ${!isDashboardRoute(this.props) && 'd-flex justify-content-around w-100'}`}>
```
Comment (Saurabh186) [inline src/components/endpoints/displayEndpoint.jsx:3225]:
```suggestion
          <div className={`innerContainer response-bottom w-100 ${!isDashboardRoute(this.props) && 'd-flex justify-content-around'}`}>
```

Comment (Saurabh186) [reply src/components/endpoints/displayEndpoint.jsx:3225]: remove w-100 in !isDashboardRoute(this.props

File: src/components/endpoints/endpoints.jsx [hunk 14]
```diff
@@ -126,9 +126,9 @@ const Endpoints = (props) => {
         </IconButtons>
       </div>
       <div className='dropdown-menu dropdown-menu-right'>
-        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleModalActionType('edit', endpointId)}> <FiEdit2 className='text-grey' size={15} /> Rename </div>
-        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleAddExampleRequest(endpointId)}> <Example className='text-grey' size={15} /> Add example </div>
-        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleDuplicate(endpointId)}> <MdOutlineContentCopy className='text-grey' size={15} /> Duplicate </div>
+        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleModalActionType('edit', endpointId)}> <FiEdit2 color='#383636' size={15} /> Rename </div>
```
Comment (Saurabh186) [inline src/components/endpoints/endpoints.jsx:129]: har jagah ek color use kar raye valibles balao phir lagao ya bna hoga syd

File: src/components/endpoints/endpoints.jsx [hunk 15]
```diff
@@ -126,9 +126,9 @@ const Endpoints = (props) => {
...
+        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleAddExampleRequest(endpointId)}> <Example color='#383636' size={15} /> Add example </div>
```
Comment (Saurabh186) [inline src/components/endpoints/endpoints.jsx:130]: ....

File: src/components/endpoints/endpoints.jsx [hunk 16]
```diff
@@ -126,9 +126,9 @@ const Endpoints = (props) => {
...
+        <div className='dropdown-item d-flex font-14 align-items-center' onClick={() => handleDuplicate(endpointId)}> <MdOutlineContentCopy color='#383636' size={15} /> Duplicate </div>
```
Comment (Saurabh186) [inline src/components/endpoints/endpoints.jsx:131]: ...

File: src/components/publishModal/publishModal.scss [hunk 17]
```diff
@@ -1,24 +1,37 @@
...
+.publish-view-site-btn {
+  .unpublish-btn {
+    border: 0.5px solid black;
   }
-  .text {
-    width: 220px;
+
+  .view-site-btn {
+    &:hover {
+      background-color: #0062cc !important;
```
Comment (Saurabh186) [inline src/components/publishModal/publishModal.scss:33]: varibles

File: src/components/tiptapEditor/tiptap.scss [hunk 18]
```diff
@@ -25,6 +24,32 @@ $scrollbar-color: #f1f1f1;
...
+    button {
+        border-radius: 4px !important;
+        margin: 0 !important;
+
+        &:hover {
+            background-color: #0000000d !important;
+        }
+    }
+
+    .create-table {
```
Comment (Saurabh186) [inline src/components/tiptapEditor/tiptap.scss:37]: class bna ke border-radius kyu likh raye use bootsrap classes

File: src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss (hunk 1 above)
Comment (ViaSocket-Git) [reply src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss:81]: Resolved

File: src/components/collectionVersions/collectionParentPages.jsx (hunk 3 above)
Comment (ViaSocket-Git) [reply src/components/collectionVersions/collectionParentPages.jsx:272]: Resolved

File: src/components/collectionVersions/collectionParentPages.jsx (hunk 4 above)
Comment (ViaSocket-Git) [reply src/components/collectionVersions/collectionParentPages.jsx:276]: Resolved

File: src/components/collections/collections.jsx (hunk 5 above)
Comment (ViaSocket-Git) [reply src/components/collections/collections.jsx:217]: Resolved

File: src/components/endpoints/endpoints.jsx (hunk 14 above)
Comment (ViaSocket-Git) [reply src/components/endpoints/endpoints.jsx:129]: Resolved

File: src/components/endpoints/endpoints.jsx (hunk 15 above)
Comment (ViaSocket-Git) [reply src/components/endpoints/endpoints.jsx:130]: Resolved

File: src/components/endpoints/endpoints.jsx (hunk 16 above)
Comment (ViaSocket-Git) [reply src/components/endpoints/endpoints.jsx:131]: Resolved

File: src/components/publishModal/publishModal.scss (hunk 17 above)
Comment (ViaSocket-Git) [reply src/components/publishModal/publishModal.scss:33]: Resolved

File: src/components/tiptapEditor/tiptap.scss (hunk 18 above)
Comment (ViaSocket-Git) [reply src/components/tiptapEditor/tiptap.scss:37]: Resolved

File: src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss [hunk 19]
```diff
@@ -71,8 +72,22 @@
...
     }
   }
+
+  @media screen and (max-width: 1280px) {
+    .main-container {
+      .form {
+        flex-direction: column;
+        gap: 20px;
+
+        .form-inputs{
```
Comment (Saurabh186) [inline src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss:82]:
```suggestion
        .form-inputs {
```spacing

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

**Review by idris-bohra — APPROVED** [review]

**Pull request merged**